lib/src/slices/data_bkg/data_bkg_service_provider.dart

```diff
@@ -0,0 +1,13 @@
+import '../api_auth_service/model/api_auth_service_account_model.dart';
+
+abstract class DataBkgServiceProvInterface {
```

Comment:
Class names should always match file names. This class should be named DataBkgServiceProviderInterface